Sources/App/Scenes/WindowScenesManager.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import Shared
+import UIKit
+
+final class WindowScenesManager {
```

Comment:
i feel like this class may be worth writing some tests around

Reply:
Updated